doc/JSON_INFO.md

```diff
@@ -401,7 +413,15 @@ Mods can modify this via "add:traits" and "remove:traits".
         [ "ARM_L", "ARM_R" ], // Overrides the above settings for those body parts
         { "bash" : 1 }        // ...and gives them those resistances instead
     ]
-]
+],
+"active" : true, //When set the mutation is an active mutation that the player needs to activate (default: false)
+"starts_active" : true, //When true, this 'active' mutation starts active (default: false, requires 'active')
+"cost" : 8, // Cost to activate this mutation. Needs one of the hunger, thirst, or fatigue values set to true. (default: 0)
+"time" : 100, //Sets the amount of (turns * current player speed ) time units that need to pass before the cost is to be paid again. Needs to be higher than one to have any effect. (default: 0)
+"hunger" : true, //If true, activated mutation increases hunger by cost. (default: false)
+"thirst" : true, //If true, activated mutation increases thirst by cost. (default: false)
+"fatigue" : true, //If true, activated mutation increases fatigue by cost. (default: false)
+"note" : "Yes, I love documentation!", // Documentation of the mutation. Obviously ignored ingame.
```

Comment:
I wanted to suggest a common way to set comments in json files. As a common one can provide benefits later (forex: somebody implements a system where json files are automatically converted to wiki pages).

Will change it.